web-api/src/business/useCases/trialSessions/serveThirtyDayNoticeInteractor.ts

```diff
@@ -138,6 +138,17 @@ export const serveThirtyDayNoticeInteractor = async (
           caseCaption: caseEntity.caseCaption,
         });
 
+        const formatCityState = ({
```

Comment:
Maybe overkill and a templated string is enough, but I did it this way because `trialSession.city` and `trialSession.state` are optional, which could theoretically lead to something silly like `, KS`.